docs/reference/concepts/hyperscaling.md

```diff
@@ -1,152 +1,246 @@
 
-Luckily, ZKPs offer a beautiful way to build a heterogeneous yet simultaneously a hyper-scalable blockchain system. This idea is known as [Fractal scaling](https://medium.com/starkware/fractal-scaling-from-l2-to-l3-7fe238ecfb4f).
-Many different ZKP chains (in the zkSync world we call them Hyperchains) are run in parallel and get their block proofs aggregated into a single final block that will be settled on L1. Each of the Hyperchains will resemble the entire system (i.e. it can have an infinite number of other Hyperchains on top of it: L3, L4, and so on).
+<div  align="center">
 
-![Hyperchains!](../../assets/images/hyperscaling5.png "Hyperchains")
+![](../../assets/images/hyperscalingIslands.png)
 
-Fractal scaling is necessary but not sufficient to achieve hyperscaling. You need one additional component:
+</div>
 
-:::tip What are Hyper-bridges?
+In order to pass information between chains, bridges based on economic incentives were introduced. These require the operators to lock funds, which can be slashed based on an arbitration process.
 
-Hyper-bridges = native bridges that enable transfers between any two Hyperchains without consuming resources on a third one.
+The potential design space is vast, and there are several potential issues. These include:
 
-:::
+1. The arbitration process is not trustless; dapp developers have to consider the risks involved.
+2. These bridges usually do not transfer real assets cross-chain; they only free up already locked funds. This makes the process expensive. However, this can be solved by using general message passing and minting synthetic assets. But the synthetic assets are less secure than the original assets and are not interchangeable with them.
+3. Chains can hard fork. This is their right, but it endangers the entire bridging ecosystem. This can be mitigated by locking funds separately for each bridge, but this makes the process much more expensive due to liquidity fragmentatio This applies to ZK-enabled bridges as well.
+4. Finally, even if the economic incentives are sound, hacks can always occur. If a vulnerability is found in the smart contracts, then every bridge in the entire ecosystem is affected. With the growth of adoption, cross-chain activity will be an indispensable part of every chain's activity. At this stage, if a bug is found, then every chain will want to hard fork to save its bridges. Bridges can only be hard forked on both sides with consensus. This means there needs to be a common social consensus for the ecosystem.
 
-Fractal scaling can always have native bridges connecting chains via underlying layers, but in this case eventually the Basechain will turn into a crossroad for most transfers and thus the central scalability bottleneck, defeating the very idea of parallel hyperscalability. As a result it won’t be possible to guarantee cheap direct transfers between users on any given two chains.
+## The Solution: Hyperchains + Hyperbridges
 
```

Comment:
I think should mention what Hyperchains are in this section. We can bring the original section ["What are hyperchains"](https://era.zksync.io/docs/reference/concepts/hyperscaling.html#what-are-hyperchains) here. It will also help with SEO 😉